docs/src/code/building-linuxcnc.adoc

```diff
@@ -5,34 +5,84 @@
 
 == Introduction
 
-This document describes how to build the LinuxCNC software and
-documentation from source.  This is primarily useful if you are a
-developer who is modifying LinuxCNC.  It can also be useful if you're
-a user who is testing developer branches, though then you also have
-the option of just installing Debian packages from the buildbot:
-http://buildbot.linuxcnc.org
+This document describes how to build the LinuxCNC software from source.
+This is primarily useful if you are a developer who is modifying LinuxCNC.
+It can also be useful if you're a user who is testing developer branches,
+though then you also have the option of just installing Debian packages from the buildbot
+(http://buildbot.linuxcnc.org) or as a regular package from your Linux
+distribution (https://tracker.debian.org/pkg/linuxcnc).
+Admittedly, this section also exists since LinuxCNC is a community effort.
+and you are encouraged to contribute to the development of LinuxCNC.
+You want to compile LinuxCNC yourself for The exchange of new developments,
+or you may be porting LinuxCNC to some new Linux disribution
+or developer reacts to you reporting a problem that you want to test
+but any such change will see no buildbot to help, or that help is delayed,
+pending someone else's review that you do not want to wait for or you are
+the only other individual with a particular hardware to test the code.
+
+Besides the programs that control your machine that are built from the source tree, 
+you can also build the same PDFs and/or HTML files that you are likely to have encountered
+online on https://linuxcnc.org/documents/.
+
+If you want to contribute to LinuxCNC but are uncertain about where to start,
+please seriously consider to contribute to the documentation.
+Everyone always finds something to improve - and if you only leave
+a "FIXME: with a comment" in the text as a reference for your self
+and others to revisit a section later. Also, translations to languages
+other than English are very likely to benefit from your scrutiny at
+https://hosted.weblate.org/projects/linuxcnc/ .
+
+== Downloading source tree
+
+The LinuxCNC project git repository is at https://github.com/LinuxCNC/linuxcnc.
+GitHub is a popular git hosting service and code sharing website.
```

Comment:
Suggest to drop the github explanation.  Seem out of place and do not really contribute to the explanation on how to download the source.

Reply:
@smoe Do you disagree or just not notice this proposal?